.backportrc.json

```diff
@@ -4,6 +4,8 @@
     "targetBranchChoices": ["v0.6"],
     "branchLabelMapping": {
       "^backport/(.+)$": "$1"
-    },    
-    "targetPRLabels": ["backport/skip"]    
+    },
+    "autoMerge": true,
```

Comment:
Will this wait for all checks to succeed?

Reply:
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/automatically-merging-a-pull-request